src/components/Navbar/SearchLocation/SearchLocation.js

```diff
@@ -1,7 +1,33 @@
-import React, { Component } from "react";
+import React, { useState, useEffect } from "react";
+import data from "./data.json";
 
-export default class SearchLocation extends Component {
-  render() {
-    return <div>[Search location]</div>;
+export default function SearchLocation() {
+  const [states, setStates] = useState([]);
+
+  // const API_KEY = "e14y0OqlWGddCHCXLC3UzLBlcKecsGrxlahVvogx";
+
+  // const stateEl = useRef();
+
+  useEffect(() => {
+    fetchListOfStates();
+  }, []);
+
+  function fetchListOfStates() {
+    setStates(data);
   }
+
+  return (
+    <div>
+      <select id="states">
+        <option value="" hidden>
+          Select a state
+        </option>
```

Comment:
Referring to this in my comment above

Reply:
When you click on the dropdown, "select a state" won't show as an option :)